### PR TITLE
ENCD-6104-fix-series-table-accession-links

### PR DESCRIPTION
**Pull request by forresttanaka**

### CODE REVIEW

File: src/encoded/static/libs/user_roles.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+     * Returns true if the current user created their own account.
+     */
+    get isUnprivileged() {
+        return this._roles.length === 1 && this._roles[0] === 'unprivileged';
+    }
+
+    /**
+     * Returns true if the current user is a submitter or admin.
```
Comment (zoldello) [inline src/encoded/static/libs/user_roles.js:54]: technically, this only checks for submitter.

File: src/encoded/static/libs/user_roles.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+     */
+    get isPrivileged() {
+        return this._roles.includes('submitter');
+    }
+
+    /**
+     * Returns true if the current user is a lab submitter.
+     */
```
Comment (zoldello) [inline src/encoded/static/libs/user_roles.js:62]: Can a submitter not also be an admin?

File: src/encoded/static/libs/user_roles.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    /**
+     * Returns true if the current user created their own account.
+     */
+    get isUnprivileged() {
+        return this._roles.length === 1 && this._roles[0] === 'unprivileged';
+    }
+
+    /**
+     * Returns true if the current user is a submitter or admin.
+     */
```
Comment (zoldello) [inline src/encoded/static/libs/user_roles.js:55]: Maybe it is just me - I could no tell what this does looking at the name. Like,  privilege to do what?

**Review by zoldello — APPROVED** [review]
Code looks good.  I recommend you look at my comments and decide if they can be dismissed or not.

File: src/encoded/static/libs/user_roles.js (hunk 1 above)
Comment (forresttanaka) [reply src/encoded/static/libs/user_roles.js:54]: If you have 'admin' in the roles, then you also have 'submitter' in the roles, so checking only 'submitter' returns true for both submitters and admins.

File: src/encoded/static/libs/user_roles.js (hunk 3 above)
Comment (forresttanaka) [reply src/encoded/static/libs/user_roles.js:55]: Just means submitter and admins, just in contrast to unprivileged (the only thing I’ve ever heard of logged-in users who have created their own accounts called) and not-logged-in users.

File: src/encoded/static/libs/user_roles.js (hunk 2 above)
Comment (forresttanaka) [reply src/encoded/static/libs/user_roles.js:62]: Meaning specifically a submitter, not an admin. If you want both, then `isPrivileged`. If you want just admins, then `isAdmin`. If you want just submitters, then `isSubmitter`.

**Pull request merged**